### PR TITLE
#moarSSL free SSL links

### PR DESCRIPTION
**Pull request by cottsak**
this is free so folks should know about it.
please don't use #CloudBleed to say something silly like "CloudFlare is not safe because it was hacked": https://www.troyhunt.com/pragmatic-thoughts-on-cloudbleed/

### CODE REVIEW

File: security.md [hunk 1]
```diff
@@ -57,9 +57,9 @@ You would need at least one domain for your website, one for your API, and one f
 
 * A third domain is needed for internal use and back office. This domain would probably be registered anonymously, so it would be a little more difficult to find.
 
-### Use SSL where possible
+### Use SSL/TLS/HTTPS everywhere!
```
Comment (itaifrenkel) [inline security.md]: this one is ok

File: security.md [hunk 2]
```diff
@@ -57,9 +57,9 @@ You would need at least one domain for your website, one for your API, and one f
 
 * A third domain is needed for internal use and back office. This domain would probably be registered anonymously, so it would be a little more difficult to find.
 
-### Use SSL where possible
+### Use SSL/TLS/HTTPS everywhere!
 
-Use SSL anywhere possible. In your website, your API, your back office servers, and if it's not too difficult even between internal services.
+Use SSL anywhere possible. In your website, your API, your back office servers, and if it's not too difficult even between internal services. You can even get away with [free SSL using CloudFlare](https://www.troyhunt.com/how-to-get-your-ssl-for-free-on-shared/) for some time before you need to pay for it.
```
Comment (itaifrenkel) [inline security.md]: AWS would give you free certs forever. What's CloudFlare value proposition here?

Comment (cottsak) [inline security.md]: > AWS would give you free certs forever

With your own domain name? You're not talking about a shared cert are you?

Comment (rjmunro) [inline security.md]: You can get free SSL from at least StartSSL, Let's Encrypt and CloudFlare, possibly more. CloudFlare's system requires using them as your DNS server, and possibly forwarding all traffic through them, which you may not want to do (although I've used it in the past, and it is a great service).

Comment (itaifrenkel) [inline security.md]: As I wrote at the bottom of the document, the vendor mentioning issue is tricky. Let's continue the discussion here https://github.com/forter/security-101-for-saas-startups/issues/30

I will close this PR now

**Comment by itaifrenkel** [comment]
Need to discuss vendor issue

**Comment by cottsak** [comment]
@itaifrenkel Rather than closing the PR and loosing what value could have been in part, why couldn't we just propose to drop the link and leave the remainder of the improvements?

**Pull request merged**